packages/scrollregion/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@zendeskgarden/container-scrollregion",
+  "description": "Containers relating to scroll region in the Garden Design System",
+  "license": "Apache-2.0",
+  "author": "Zendesk Garden <garden@zendesk.com>",
+  "homepage": "https://garden.zendesk.com/",
+  "repository": "https://github.com/zendeskgarden/react-containers",
+  "bugs": {
+    "url": "https://github.com/zendeskgarden/react-containers/issues"
+  },
+  "version": "1.3.8",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.esm.js",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "../../utils/scripts/build.sh"
+  },
+  "sideEffects": false,
+  "types": "dist/typings/index.d.ts",
+  "dependencies": {
+    "@babel/runtime": "^7.8.4",
+    "@types/lodash.debounce": "^4.0.6",
+    "lodash.debounce": "^4.0.8",
+    "react-merge-refs": "^1.1.0"
```

Comment:
I've updated `@types/lodash.debounce` to `devDependencies`. Was there another package that needed to be moved?

I noticed `@babel/runtime` as a dependency that could (maybe?) be moved to `devDependencies`, but also noticed this for the other packages also.

Reply:
I'm not sure what the driving factor was for making `@babel/runtime` a direct dependency. Would probably require research.

Reply:
I'm going to leave `@babel/runtime` since the docs instructions [indicate](https://babeljs.io/docs/en/babel-runtime) by default it's a direct dependency (as opposed to something like a babel [plugin](https://babeljs.io/docs/en/babel-preset-env#install)).